### PR TITLE
Feature/ngr lookup

### PR DESCRIPTION
**Pull request by timstone**

### CODE REVIEW

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
I've completed PR #5 to get the admin nav bar section in. That should hopefully clean up some of the stuff around working with the menu.

I'd suggest getting that in first  i.e. complete #5, merge it in, rebase this branch on `master`, and then update it to use the section.

We can then go from there.

**Comment by timstone** [comment]
@Cruikshanks rebased with master

File: features/backoffice_site_address.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@backoffice
+Feature: Back office user registers a customers new waste exemption activity
```
Comment (Cruikshanks) [inline features/backoffice_site_address.feature]: Could we make the feature description more relevant to the test i.e. try and put something in here as to why we need to capture the EA admin area

File: features/backoffice_site_address.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  Scenario Outline: Registration by a NCCC user using a postcode for site address - NGR and area is added to registration
+     When I complete a registration using a <postcode> for site address
+     Then I will be able to check what the National Grid Reference <ngr> is for the site
+     And I will be told what <area> that site is located in
+
+     Examples:
+     | postcode | ngr           | area   |
+     | BS1 5AH  | ST5813072687  | wessex |
+
+
```
Comment (Cruikshanks) [inline features/backoffice_site_address.feature]: Remove the extra lines and ensure we have a carriage return at the end of document.

File: features/backoffice_site_address.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  I want to register a waste exemption
+  So that I can register exemptions for customers over the phone
+
+  Background:
+    Given I am an internal user
+      And I have a valid username and password
+
+  @happy_path
+  Scenario Outline: Registration by a NCCC user using a postcode for site address - NGR and area is added to registration
+     When I complete a registration using a <postcode> for site address
```
Comment (Cruikshanks) [inline features/backoffice_site_address.feature]: Just sort out the grammar. Perhaps...
```gherkin
When I complete a registration using postcode <postcode> for the site address
```

File: features/backoffice_site_address.feature [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+     Then I will be able to check what the National Grid Reference <ngr> is for the site
```
Comment (Cruikshanks) [inline features/backoffice_site_address.feature]: Just sort out the grammar. Perhaps...

```gherkin
Then I will see the site's National Grid Reference has been automatically found to be <ngr>
```

File: features/backoffice_site_address.feature [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+     And I will be told what <area> that site is located in
```
Comment (Cruikshanks) [inline features/backoffice_site_address.feature]: Just sort out the grammar. Perhaps...

```gherkin
And I will see the EA admin area is set to <area>
```

**Review by Cruikshanks — CHANGES_REQUESTED** [review]
Just some tidying up to do in the `.feature` file.

Also looks like we merged in the admin section stuff rather than rebasing on `master`. No biggy, but I can show you how to **rebase** instead, and what the difference is next time.

**Comment by timstone** [comment]
@Cruikshanks I've updated features as discussed.

**Review by Cruikshanks — APPROVED** [review]

**Comment by Cruikshanks** [comment]
When you do the merge in Github @timstone can you use the drop down and select **Squash & merge**. This will ensure we clean up the commits on this branch before merging them into `master`. Come give me a shout if you need a hand.

**Pull request merged**